edx_proctoring/templates/proctoring/seq_timed_exam_ready_to_submit.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+<div class="sequence timed-exam completed" data-exam-id="{{exam_id}}">
+  <h3>
```

Comment:
correct, but if you have some 'drive-by comments' I can see if we can do it, but I think the whole unproctored timed exams still needs to go through some Product/UX conversations.